packages/allure-cypress/src/reporter.ts

```diff
@@ -90,7 +120,7 @@ export const allureCypress = (on: Cypress.PluginEvents, config?: AllureCypressCo
           const currentStep = currentSteps[currentSteps.length - 1];
 
           parameter.forEach(({ name, value, excluded, mode }) => {
-            currentTest.parameter(name, value, {
+            (currentStep || currentTest).parameter(name, value, {
```

Comment:
Yes, I did, because we had recieved a PR with the functionaly from the community. So users expect, that parameters will be applied to the current step, when it exists.